Documentation.md

```diff
@@ -34,31 +34,31 @@ If you first time running WSL or Linux installation, follow these steps:
 `sudo apt-get install lld`
 ___________________________________________________________________
 
-For compiling bootloader,
+To compile the bootloader (gnu-efi) you will need to run the following commands in your Terminal (Linux) or using WSL/WSL2 on Windows:
 
 `cd gnu-efi`
 
 `make bootloader`
 
 ________________________________________________________________
 
-For compiling kernel,
+To compile the ringOS kernel you will need to run the following commands in your Terminal (Linux) or using WSL/WSL2 on Windows:
 
 `cd kernel` or (if you are in gnu-efi folder) `cd ../kernel`
 
 `make kernel`
 
 ______________________________________________________________
 
-For running image,
+To run ringOS within Qemu on Linux or Windows:
 
-`make buildimg` [kernel folder]
```

Comment:
Where is make `make buildimg`

Reply:
It was moved down under the Windows "run2.bat" instruction to keep the linux instructions together.

Reply:
I didn't see

Reply:
I saw it